frontend/app/components/person-edit.js

```diff
@@ -196,6 +201,8 @@ export default ApplicationComponent.extend(EKMixin, {
         person.rollbackAttributes();
       }
 
+      this.set("person.company", this.get("callBackCompany"));
```

Comment:
This doesn't seem to be working, if you abort the edit it still stays the new value